backend/api/tests/test_views.py

```diff
@@ -1,14 +1,17 @@
 import pytest
 import vcr
-import requests
+from django.test import Client
+
+client = Client()
 
 @vcr.use_cassette()
-def test_api_root():
-    response = requests.get('http://localhost:8000/api/v1/')
+def test_api_root(client):
+    response = client.get('/api/v1/')
     assert response.status_code == 200
-    assert 'businesses' in response.text
 
-@vcr.use_cassette()
-def test_businesses_endpoint():
-    response = requests.get('http://localhost:8000/api/v1/businesses/')
+
+@vcr.use_cassette(record_mode='new_episodes')
```

Comment:
The cassette this creates shows both the request to the businesses endpoint and the request to Socrata. However, it includes the app token in the headers of the request to Socrata. I tried adding a `filter_headers=['X-App-token']` here, but it doesn't seem to do anything... hmmm